cypress/e2e/homepageSettings.js

```diff
@@ -1,6 +1,6 @@
 describe('Homepage Settings', () => {
   beforeEach(() => {
-    cy.localLogin({ name: 'MINT' });
+    cy.localLogin({ name: 'ANON' });
```

Comment:
Was there something in this change that made this necessary?

Reply:
Yes, the seed data for MINT adds view customization for the MINT user. This cypress test only expects that the user doesn't have any customizations. `ANON` doesn't have any seed data associated.

Reply:
Ah! Might be worth just a quick `// Using ANON since that user has no customized views` in case anyone ever adds data and sees that test break! But thanks for the explanation!